resources/sass/app.scss

```diff
@@ -1009,3 +1009,144 @@ nav {
   letter-spacing: -0.28px;
   text-transform: uppercase;
 }
+
+
+.page-dropdown-menu {
```

Comment:
css from screen builder is already included in core at `bootstrap.js` so this style will not be needed here.